src/eu/ldbc/semanticpublishing/properties/Configuration.java

```diff
@@ -133,6 +136,9 @@ public Configuration() {
 		properties.setProperty(VALIDATE_HISTORY_PLUGIN, "false");
 		properties.setProperty(HISTORY_QUERIES, "");
 		properties.setProperty(LOG_FAILED_HISTORY_QUERIES, "false");
+		properties.setProperty(PRELOAD_ANALYTICAL_QUERY_RESULTS, "true");
```

Comment:
If this is true and the file is missing, the query will be run right?

Reply:
If its true, and the file is missing it will analyze the DB, save the results to file and continue as normal